aiosqlite/core.py

```diff
@@ -72,10 +72,15 @@ def __init__(
                 DeprecationWarning,
             )
 
-    def _stop_running(self):
```

Comment:
def _stop_running(self):

Reply:
>     def _stop_running(self):

All review points are the same type: verbatim copy of a line change.

If you are not some kind of AI, would you mind expanding with clarifying comments ?